### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #1: Create contributing guidelines
Create CONTRIBUTING.md with contributing guidelines.

<br />

Things we would like to achieve with our guidelines:
 - Beginner friendly and easy to follow
 - Clear and precise instructions
 - Streamlined process

Assigning this to myself, but please comment if there is anything I should take into consideration before creating a PR.
All ideas and suggestions are welcome! ^^

### PR DESCRIPTION
**Pull request by marko-radosavljevic**
Closes #1

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first! :tada: 
+
+
+## Getting started
+
+1. Create your own fork of the repository.
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate.
+3. To comply with our style guidelines use `./gradlew spotlessApply`. PRs should not include commits with autostyle runs. Instead, fix any style violations before committing changes.
```
Comment (marko-radosavljevic) [inline CONTRIBUTING.md:34]: Obviously, this is assuming the tech stack we will use, so I guess we will just edit it later if this prediction is wrong.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+6. Each commit name and message should be clear, concise and informative: Please consider checking these resources: [writing a commit message](https://chris.beams.io/posts/git-commit/) and [writing a good commit message](https://dev.to/chrissiemhrk/git-commit-message-5e21)
+7. Each commit should be small, contained and should solve only one problem.
+8. Pull request should not include any merge commits. To update the branch of your pull request use `git pull --rebase`.
+
+
+## Community
+
+You can chat with the TJ-Bot users and devs in our [discord server](https://discord.gg/UfasCQNv)!
+
+#### If you find a security vulnerability, do NOT open an issue. Instead, consult the instructions in the [Security Policy](https://github.com/Together-Java/TJ-Bot/blob/master/SECURITY.md)
```
Comment (marko-radosavljevic) [inline CONTRIBUTING.md:46]: SECURITY.md doesn't exist, but it should in the future. Should we leave this here?

Comment (Tais993) [inline CONTRIBUTING.md:46]:
```suggestion
#### If you find a security vulnerability, do NOT open an issue. Instead, consult the instructions in the [Make in progress](https://github.com/Together-Java/TJ-Bot/blob/master/SECURITY.md)
```
Rename it so it's clear till it's added?

Comment (Zabuzard) [reply CONTRIBUTING.md:46]: Yeah. Everything is WIP now. Lets take it and edit things later.

Comment (Zabuzard) [reply CONTRIBUTING.md:46]: I would just leave it as it is and create the file later. Its just a red-link until then, which is fine.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Zabuzard) [reply CONTRIBUTING.md:34]: Yeah. Everything is WIP now. Lets take it and edit things later.

**Review by Zabuzard — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Tais993) [reply CONTRIBUTING.md:46]: If it's displayed as a red-link sure

**Review by Tais993 — APPROVED** [review]
Looks good

**Comment by marko-radosavljevic** [comment]
Want to wait a bit more before merging this, since not many people seen it, and I would like more opinions. ^^

Docs usually get forgotten after some time, so I want to make sure we have a decent starting point.

**Pull request merged**